Handbook: Explaining how startup programs work (#12164)
PostHog/posthog.com: contents/handbook/brand/startups.md

showTitle: true
---

We run two special programs for early-stage teams. The primary place for discussing both programs is the [#project-startups-and-yc](https://posthog.slack.com/archives/C088RSQKH2T) channel in Slack.

| Feature                     | Startups                      | Y Combinator                     |
|----------------------------|-------------------------------|----------------------------------|
| Eligibility                | <2 years old, <$5M raised     | Must be in YC (any batch)        |
| Credit                     | $50,000 for 12 months         | $50k (recent batches) or $25k    |

> | BAA eligible?              | ❌ Cannot use credit for BAA  | ✅ Can use credit for BAA        |

Reviewer: Worth adding explicit row here for being able to spend it on platform add-ons - AFAIK it's "yes" for both (BAA is an exception) - it's a common support question even though it's mentioned somewhere in the docs

Author: ```suggestion
| Can use credit for add-ons?          | ⚠️ Yes, but cannot use credit for BAA in Boost add-on | ✅ Yes, and can use credit for BAA in Boost add-on        |
```

> | Founder merch              | Welcome pack (max 4)          | Different welcome pack (max 4)   |

Reviewer: We only allow 1 pack for startup program applicants

| Community                  | —                             | Private YC founder Slack         |
| Apply via…                 | [Startup page](/startups)     | [Secret YC page](https://app.posthog.com/startups/yc) |

## PostHog for Startups

Any company that is <2 years old and has raised less than $5M in funding is eligible to apply and claim the following:

- $50,000 in PostHog credits (valid for 12 months)

> - Up to 4 unique welcome packs for founders (see [Welcome merch](#welcome-merch))

Reviewer: As above - just 1 (at least in current implementation) for startups

Author: ```suggestion
- One unique welcome pack for founders
```

We track all PostHog for Startups applications in [this Zapier table](http://tables.zapier.com/app/tables/t/01JRARGWTSDYCGNS12HXN3B6DY).

## PostHog for Y Combinator

This program is similar to our startup program but has some key differences for YC teams. Teams can be in any YC batch, with any amount of funding raised, and can claim the following:

- A variable amount of PostHog credit:
  - $50,000 for teams in the current, upcoming, or last 4 YC batches
  - $25,000 for older batches
- Up to 4 unique founder merch packs (different from the startup program)
- Access to [our private YC Founder Slack](https://posthog.slack.com/archives/C04J1TJ11UZ)
- Partner benefits with Speakeasy and Mintlify (50% off for 6 months)
- A monthly founder newsletter

> ✅ Credits **can** be used to claim a BAA under the Boost plan.

YC teams must apply via [our secret YC page](https://app.posthog.com/startups/yc). If we can’t immediately verify they’re part of YC, we ask for a screenshot from Bookface to prove their eligibility.

We track all PostHog for YC applications in [this Zapier table](https://tables.zapier.com/app/tables/t/01JRCYMWYAJNP3K0B6GTYKKBQB).

> ## What Happens After Companies Apply

Author: ```suggestion
## What happens after companies apply?
```

## Monthly Newsletter

We send a short, founder-focused newsletter once per month to all program participants. This is handled as a Customer.io broadcast using [a prebuilt template](/handbook/brand/email-comms).

## Credit usage

Credits can be used for almost all PostHog products and add-ons, including [platform add-ons](/platform-addons).

- **Startups**: ❌ Cannot use credits toward a BAA due to legal risk.
- **YC teams**: ✅ Can use credits for a BAA under the Boost plan.

Credits are valid are not transferable, and don’t carry over or convert to cash. They are valid for 12 months and that timer begins at application. Once expired or fully used, teams are moved to standard billing.

## Partners

We currently partner with:

- **Mintlify** — 50% off for 6 months
- **Speakeasy** — 50% off for 6 months

Discount codes are sent in the welcome email after signup.

If users run into issues with redemption, we can help liaise — though all offers are ultimately at partner discretion.

Contacts:
- Mintlify: Tiffany Chen  
- Speakeasy: Nolan Di Mare Sullivan

We previously offered DigitalOcean credits ($25k), but this was retired in Q2 2025.

> ## Program Extensions

Author: ```suggestion
## Program extensions
```
